examples/basic_interdomain/README.md

```diff
@@ -0,0 +1,16 @@
+# Basic examples
+
+Contain basic setup for NSM that includes `nsmgr`, `forwarder-vpp`, `registry-k8s`. This setup can be used to check mechanisms combination or some kind of NSM [features](../features).
```

Comment:
@MarinaShustowa, I suppose this explanation is not correct and is not full.

Reply:
Thank you, fixed.